server/test/queries/updatePersonalDataQuery.test.js

```diff
@@ -0,0 +1,40 @@
+const tape = require('tape');
+
+const { updatePersonalDataQuery } = require('./../../database/queries/updatePersonalData');
+const { buildDb, buildFakeData, buildStaticData } = require('./../../database/config/build');
+
+tape('Query - Update User Personal Data || Valid', (e) => {
+  const userId = 1;
+  const firstName = 'Khader';
+  const lastName = 'Murtaja';
+  const email = 'aaa@bbb.com';
+  buildDb()
+    .then(() => buildStaticData())
+    .then(() => buildFakeData())
+    .then(() => {
+      updatePersonalDataQuery(firstName, lastName, email, userId)
+        .then((result) => {
+          e.equal(result.rows[0].id, 1, 'Personal Data Have Updated in Database !!');
```

Comment:
you should test if the data updated link : 
```javascript
e.equal(result.rows[0].firstName, 'Khader', 'Personal Data Have Updated in Database !!');

```

Reply:
@knanahassouna1, Done